core/src/rpc.rs

```diff
@@ -109,6 +110,7 @@ pub struct JsonRpcConfig {
     pub enable_bigtable_ledger_storage: bool,
     pub enable_bigtable_ledger_upload: bool,
     pub max_multiple_accounts: Option<usize>,
+    pub supported_indexes: Vec<IndexType>,
```

Comment:
```suggestion
    pub account_indexes: Vec<AccountIndex>,
```

Reply:
Or if renaming `IndexType` is too much it would be good to only import `accounts_index` and qualify this and other usage to `accounts_index::IndexType`.

Reply:
No problem to rename!

Reply:
Done in https://github.com/solana-labs/solana/pull/14212/commits/d204a08432c3936cd1450cf16e1a67236e758461